SDW/alc1308.asl

```diff
@@ -0,0 +1,587 @@
+ /** @file
+  The definition block in ACPI table for rt5682 device under I2C0 Controller
+
+Copyright (c) 2015, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+**/
+
+DefinitionBlock (
+  "",
+  "SSDT",
+   2,
+  "INTEL",
+  "1308Tabl",
+  0x1000
+  )
+{
+External(\_SB.PCI0.HDAS.SNDW, DeviceObj)
+External(SDW2, UnknownObj)
+
+Scope (_SB.PCI0.HDAS.SNDW) {
+	Device (RTKX)
+        {
+            Name (_ADR, 0x000220025D130800)  // _ADR: Address

```

Comment:
you sure about the _ADR?
should be link 2, rtk ID is 025D and part 1308 so there's an extra 2?
also the links start at zero, link2 is the 3rd master.

Reply:
> you sure about the _ADR?
> should be link 2, rtk ID is 025D and part 1308 so there's an extra 2?

@plbossart The first 2 is sdw version, should we remove it?

> also the links start at zero, link2 is the 3rd master.

Yes, it is connected to 3rd master on my platform.